contrib/builder/arm/deb/debian-jessie/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ioft/armhf-debian:jessie
+
+RUN apt-get update && apt-get install -y apparmor bash-completion btrfs-tools build-essential curl ca-certificates debhelper dh-apparmor dh-systemd git libapparmor-dev libdevmapper-dev libltdl-dev libsqlite3-dev libsystemd-journal-dev --no-install-recommends && rm -rf /var/lib/apt/lists/*
+
+ENV GO_VERSION 1.5.1
+RUN curl -fSL "https://github.com/hypriot/golang-armbuilds/releases/download/v${GO_VERSION}/go${GO_VERSION}.linux-armv7.tar.gz" | tar xzC /usr/local
```

Comment:
@thaJeztah to speed up the build process we'd recommend using pre-compiled Go versions, and it's a lot easier than building Go 1.5.1 from source too. There is already an open issue at https://github.com/golang/go/issues/13066 and we're expecting to get officially built pre-compiled Go tarballs for ARM later, but this is scheduled for Go 1.6. This been said, we'd like to take the easy and fast approach and using our own pre-compiled versions of Go for ARMv7. BTW, this build process is already documented in that specific repo. And as soon as the official Go tarballs for ARM are available, we'll change this to the official downloads.